pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/ddp/participant/ParticipantDto.java

```diff
@@ -35,6 +36,14 @@ public void setParticipantId(int participantId) {
         this.participantId = Optional.of(participantId);
     }
 
+    public int getParticipantIdOrThrow() {
+        return getParticipantId().orElseThrow(() -> new DsmInternalError("Participant ID cannot be null"));
```

Comment:
This should not happen, but in case we ever reached here can we add more to this log ? maybe logging the ddp_participant_id